automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -xe
```

Comment:
u might want to add -u (see comment about it below),
and -E because u use '(' subshell

Reply:
Done